# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieService.cpp
# Commit: da9446e06954
# Full Hash: da9446e06954e959c0bcfbec51f299d3132f5fbc
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-18 04:27:40
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r 20708fe16d2a -r da9446e06954 netwerk/cookie/CookieService.cpp
--- a/netwerk/cookie/CookieService.cpp	Tue Dec 17 21:44:37 2024 +0000
+++ b/netwerk/cookie/CookieService.cpp	Tue Dec 17 21:44:38 2024 +0000
@@ -1814,5 +1814,14 @@
   return NS_OK;
 }
 
+NS_IMETHODIMP
+CookieService::TestGet3PCBExceptions(nsTArray<nsCString>& aExceptions) {
+  aExceptions.Clear();
+
+  mThirdPartyCookieBlockingExceptions.GetExceptions(aExceptions);
+
+  return NS_OK;
+}
+
 }  // namespace net
 }  // namespace mozilla